python/lsst/pipe/tasks/deblendCoaddSourcesPipeline.py

```diff
@@ -213,12 +229,17 @@ def run(self, coadd, mergedDetections, idFactory):
         return Struct(measureCatalog=sources)
 
 
-class DeblendCoaddSourcesMultiTask(DeblendCoaddSourcesBaseTask):
+class DeblendCoaddSourcesMultiTask(PipelineTask):
```

Comment:
Do we want to rename this to `DeblendCoaddSourcesTask` now that there is no `DeblendCoaddSouresSingleTask`?

Reply:
I don't think it's worth the additional disruption right now (though if we did rename it, I'd want to get `Sources` out of the names of all of the coadd-processing tasks, too).